src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -1610,7 +1610,7 @@ void Unit::HandleEmoteCommand(uint32 emoteId)
     damageReduction /= (1.0f + damageReduction);
 
     RoundToInterval(damageReduction, 0.f, 0.75f);
-    return uint32(std::ceilf(std::max(damage * (1.0f - damageReduction), 0.0f)));
+    return uint32(ceilf(std::max(damage * (1.0f - damageReduction), 0.0f)));
```

Comment:
this one should not be in this PR

Reply:
Correct, I only added it in to fix the CI build. I'll take it out to reduce confusion.

Reply:
Misspoke in my comment for 0f0f3da. Without the fix, the build is outright broken. When I said it worked locally I must have gotten my wires crossed and been thinking of a pre-3.3.5 merge build.

Reply:
3.3.5 has the build-fix now